instrumentation/jdbc/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/jdbc/PreparedStatementInstrumentation.java

```diff
@@ -51,6 +53,11 @@ public static void onEnter(
         @Advice.This PreparedStatement statement,
         @Advice.Local("otelContext") Context context,
         @Advice.Local("otelScope") Scope scope) {
+      // prevent recursive Statement calls
```

Comment:
can you expand on this just a bit? at first glance I didn't understand why this was different from our nested CLIENT span suppression